core/arch/x86/decode_table.c

```diff
@@ -1976,6 +1976,24 @@ const instr_info_t * const op_instr[] =
 #define vsiby    REQUIRES_VSIB_YMM
 #define vsibz    REQUIRES_VSIB_ZMM
 
+/* AVX-512 tupletype attributes. They are moved into the upper half
+ * of the instr_info_t flags.
+ */
+#define ttnone (TUPLE_TYPE_NONE * 65536)
```

Comment:
Also, please define the bit ranges like we do for dr_pred_type_t: see PREFIX_PRED_BITS, PREFiX_PRED_BITPOS, PREFIX_PRED_MASK.  (That's where not seeing the code that is using it makes it harder to review.)

Reply:
Done.